CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -0,0 +1,189 @@
+# CHANGELOG-0.1
```

Comment:
For all changelogs, since they're generated, it's better to add this to the `.gitattributes` as generated so that the diff is hidden for every release. https://github.com/github/linguist/#generated-code